apps/webapp/src/state/send.ts

```diff
@@ -109,10 +111,11 @@ const planWitnessBuildBroadcast = async ({ amount, recipient, selection, memo }:
 
   if (!plan) throw new Error('no plan in response');
 
+  // Remove `authorizationData` after modifying protobuf definition
   const { data: authorizationData } = await custodyClient.authorize({ plan });
   if (!authorizationData) throw new Error('no authorization data in response');
 
-  const { transaction } = await viewClient.witnessAndBuild({
+  const { transaction } = await viewClient.authorizeAndBuild({
     transactionPlan: plan,
     authorizationData,
   });
```

Comment:
this is the source of the double popup - the custody service authorize impl is where the popup is requested. we call it here in send.ts, and then also in the impl

https://github.com/penumbra-zone/web/blob/0104733f44e08d1d27d1e9c7a110da5ddcf870b1/packages/router/src/grpc/view-protocol-server/authorize-and-build.ts#L13-L17

we could simply remove the authorize call here in send.ts, and pass an empty authorizationdata to authorizeandbuild.

Reply:
cc penumbra-zone/penumbra#3618

Reply:
I recalled protobuf fields are optional by default, and removing the `AuthorizeData` field entirely seemed to work.